Source/Core/Core/HW/Wiimote.cpp

```diff
@@ -3,10 +3,15 @@
 // Refer to the license.txt file included.
 
 #include "Core/HW/Wiimote.h"
+
 #include "Common/ChunkFile.h"
+#include "Common/CommonTypes.h"
+
```

Comment:
Why separate all these with newlines (other than Core/HW/Wiimote.h)? clang-format will sort them the same way as you did manually.